typescript/src/shared/disputes/parameters.ts

```diff
@@ -0,0 +1,65 @@
+import {z} from 'zod';
+import type {Context} from '@/shared/configuration';
+
+export const updateDisputeParameters = (_context: Context = {}) =>
+  z.object({
+    dispute: z.string().describe('The ID of the dispute to update'),
+    evidence: z
+      .object({
```

Comment:
I picked 3 fields of the many fields available here, sort of based on best/random judgment. We should add a file upload tool maybe and then we can add other properties here